src/toil_lib/tools/variant_annotation.py

```diff
@@ -0,0 +1,106 @@
+import os
+import tarfile
+
+from toil_lib.programs import docker_call
+
+
+def gatk_genotype_gvcfs(job, gvcf_ids, ref, fai, ref_dict, annotations=None, emit_threshold=10.0,
+                        call_threshold=30.0, unsafe_mode=False):
+    """
+    Runs GenotypeGVCFs on one or more gVCFs generated by HaplotypeCaller.
+
+    :param JobFunctionWrappingJob job: passed automatically by Toil
+    :param dict gvcf_ids: Dictionary of GVCF FileStoreIDs {sample identifier: FileStoreID}
+    :param str ref: FileStoreID for the reference genome fasta file
+    :param str fai: FileStoreID for the reference genome index file
+    :param str ref_dict: FileStoreID for the reference genome sequence dictionary
+    :param list[str] annotations: Optional list of GATK variant annotations. Default: None.
+    :param float emit_threshold: Minimum phred-scale confidence threshold for
+                                 a variant to be emitted. GATK default: 10.0
+    :param float call_threshold: Minimum phred-scale confidence threshold for
+                                 a variant to be called. GATK default: 30.0
+    :param bool unsafe_mode: If True, runs gatk UNSAFE mode: "-U ALLOW_SEQ_DICT_INCOMPATIBILITY"
+    :return: VCF FileStoreID
+    :rtype: str
+    """
+    job.fileStore.logToMaster('Running GATK GenotypeGVCFs')
```

Comment:
I might move this later and list the sample IDs you are running on (or do we just have filestore IDs here?) and the optional values (e.g., thresholds, annotations).